client/my-sites/importer/importer-blogger.jsx

```diff
@@ -48,14 +49,27 @@ class ImporterBlogger extends React.PureComponent {
 				}
 			),
 			uploadDescription: this.props.translate(
-				'Upload a {{b}}%(importerName)s export file{{/b}} to start importing into {{b}}%(siteTitle)s{{/b}}.',
+				'Upload a {{b}}%(importerName)s export file{{/b}} to start ' +
+					'importing into {{b}}%(siteTitle)s{{/b}}. If you want to know how to create the export file, ' +
+					'our {{inlineSupportLink/}} can help.',
 				{
 					args: {
 						importerName,
 						siteTitle: this.props.site.title,
 					},
 					components: {
 						b: <strong />,
+						inlineSupportLink: (
+							<InlineSupportLink
+								supportPostId={ 66764 }
+								supportLink={
+									'https://en.support.wordpress.com/import/coming-from-blogger/#import'
+								}
+								text={ importerName + ' ' + this.props.translate( 'import guide' ) }
```

Comment:
We may have issues in other locales where it's expected to read with a sentence structure of, for example, 'import guide Blogger' - I can't give a solid example, but someone from editorial might be able to.
I'd err on the safe side and build `importerName` in to the `translate()` call as an `arg`.

Reply:
Ah very good point, thanks, I was wondering if that was going to be OK. I'll have another go.